MusicTaste/MusicTaste/View Controllers/LoginViewController.m

```diff
@@ -0,0 +1,59 @@
+//
+//  LoginViewController.m
+//  MusicTaste
+//
+//  Created by Aman Abraham on 7/6/22.
+//
+
+#import "LoginViewController.h"
+#import <Parse/Parse.h>
+
+@interface LoginViewController ()
+@property (weak, nonatomic) IBOutlet UITextField *passwordField;
+@property (weak, nonatomic) IBOutlet UITextField *usernameField;
+
+@end
+
+@implementation LoginViewController
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    // Do any additional setup after loading the view.
+}
+- (IBAction)loginbutton:(id)sender {
+    [self loginUser];
+}
+- (IBAction)registerbutton:(id)sender {
+    [self registerUser];
+    
+}
```

Comment:
As a best practice, use camelCase consistently, e.g. `loginButton` and `registerButton`.
Also, try to format your code nicely by leaving a single blank line between functions, while removing any unnecessary blank lines.

```
- (IBAction)loginButton:(id)sender {
    [self loginUser];
}

- (IBAction)registerButton:(id)sender {
    [self registerUser];
}
```